CODE_OF_CONDUCT.md

```diff
@@ -68,9 +68,11 @@ members of the project's leadership.
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+available at [Contributor Covenant Code of Conduct][coc].
 
-[homepage]: https://www.contributor-covenant.org
+For answers to common questions about this code of conduct, see the
+[Contributor Covenant FAQ][faq]
 
-For answers to common questions about this code of conduct, see
-https://www.contributor-covenant.org/faq
+[homepage]: https://www.contributor-covenant.org
```

Comment:
Nit: maybe `cc-homepage` or `cc_homepage`?

Reply:
Good call!
Added in 7f3f110